### ISSUE TITLE
Galactic Season 5 potential armour mod bug

### ISSUE THREAD

**Issue by Isafdor** [rfelleto/SWTOR-bug-tracker#13]
**Please, provide a clear description what the bug is:**
The gear and weapons in the Galactic Season 5 reward track aren't moddable like the previous Galactic Seasonal rewards.

**Steps to reproduce the behavior with as much detail as possible:**

1. Obtain or preview a Galactic Season 5 piece of gear or weapon
2. Right click the item to check its mods
3. You are only able to apply dye modules or colour crystals instead of enhancements/armour/hilts/barrels etc

**Forum Post and IDs of affected creatures, items, quests or spells with a link to the relevant page:**

https://forums.swtor.com/topic/931061-galactic-season-5-armour-mod-bug/#comment-9776737

**Expected behavior. Describe how it should work:**
Upon right clicking the gear you should have hilt/armouring, mods and enhancement options.
I've included some screenshots of the old seasonal gear being fully moddable, alongside the new seasonal gear only allowing colour crystals and dye.
![bug1](https://github.com/rfelleto/SWTOR-bug-tracker/assets/45134576/b87a5670-e1a4-411f-a167-cb61333b9058)
![bug2](https://github.com/rfelleto/SWTOR-bug-tracker/assets/45134576/d4405058-8cf3-4026-957e-c6fec896a79d)
![bugmaybe](https://github.com/rfelleto/SWTOR-bug-tracker/assets/45134576/395c1df5-011c-4ca1-8c31-0dcdb81a699d)
![bugmaybe2](https://github.com/rfelleto/SWTOR-bug-tracker/assets/45134576/56418a42-9a55-48bd-aa85-5745d71c07a5)
![example1](https://github.com/rfelleto/SWTOR-bug-tracker/assets/45134576/2607529d-5d37-42c3-ac28-b2fa64ef5fb3)
![example2](https://github.com/rfelleto/SWTOR-bug-tracker/assets/45134576/787357bc-8e71-4400-885a-e6b2afe1dae9)

**Comment by rfelleto** [comment]
I agree. The augment slot should have been removed if they are going full cosmetic with these new sets.

**Comment by Isafdor** [comment]
Additional screenshots showing that the gear can be fully augmented with legendary augments.
![newbug1](https://github.com/rfelleto/SWTOR-bug-tracker/assets/45134576/b19c66e3-bf5f-488f-a92e-0b15dd1e404f)
![newbug2](https://github.com/rfelleto/SWTOR-bug-tracker/assets/45134576/e884f95b-e172-4099-b41a-456776efd1bd)
![newbug3](https://github.com/rfelleto/SWTOR-bug-tracker/assets/45134576/de15b064-6b88-4062-93a4-1935f39680eb)
![newbug4](https://github.com/rfelleto/SWTOR-bug-tracker/assets/45134576/83924a83-c6f3-4118-a4a4-b233f0e89ac9)
